samples/sample-04/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.0-all.zip
```

Comment:
Looks like there is already Gradle `5.1`. Also no point to force everyone to download `all`. The `bin` us fully enough for this kind of projects.